testing/softhsm/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Frank Felhoffer <silveraid@hackme.ca>
+# Maintainer: Frank Felhoffer <silveraid@hackme.ca>
+pkgname=softhsm
+pkgver=2.3.0
+pkgrel=0
+pkgdesc="cryptographic store accessible through a PKCS #11"
+url="http://www.softhsm.org/"
+arch="all"
+license="BSD2"
+depends="openssl"
+makedepends="openssl-dev automake autoconf libtool file"
+install=""
+subpackages="$pkgname-doc"
+source="softhsm-$pkgver.tar.gz::https://github.com/opendnssec/SoftHSMv2/archive/$pkgver.tar.gz"
+builddir="$srcdir/SoftHSMv2-2.3.0"
+
+prepare() {
+        cd "$builddir"
+        sh autogen.sh || return 1
+}
+
+build() {
+        cd "$builddir"
+        ./configure \
+                --prefix=/usr \
+                --sysconfdir=/etc \
+                --localstatedir=/var \
+                || return 1
+        make -j8 || return 1
```

Comment:
we should not specify `-j8` to make. The build server should have a good number set via `MAKEFLAGS` env var. Some builder will have higher number and some will have lower.

Reply:
Removed.